src/components/calcite-inline-editable/resources.ts

```diff
@@ -4,4 +4,4 @@ export const TEXT = {
   intlConfirmChanges: "Save"
 };
 
-export const transitionProperties = ["padding-left", "padding-right"];
+export const transitionProperty = "border-top-color";
```

Comment:
Nitpick: I think we will need a name that communicates how this prop is used. Currently, it's hard to associate that this prop is tied to the end of the open/close transition.

Reply:
Agreed. Should it be named according to the event? Like `editingCancelTransitionProp`?

Reply:
Maybe we can refactor this part when we have a utility for it?

Reply:
Sure, no rush on this.